openerp/addons/base/res/res_partner.py

```diff
@@ -387,6 +393,18 @@ def onchange_state(self, state_id):
             return {'value': {'country_id': state.country_id.id}}
         return {'value': {}}
 
+    @api.onchange('image')
+    def onchange_email(self):
+        if not self.image and self.email:
```

Comment:
So basically the gravatar should be the image's default value when an email is set? Isn't there a better way to say that? /cc @rco-odoo maybe a default value returning nothing if email isn't set (so it doesn't return anything during default_get but does if creating a partner without having specified an image, would that work?)

It's odd to do that by default on all partners though.